core/src/test/java/org/elasticsearch/index/mapper/CopyToMapperTests.java

```diff
@@ -149,7 +149,9 @@ public void testCopyToFieldsInnerObjectParsing() throws Exception {
                 .startObject("foo").startObject("bar").field("baz", "zoo").endObject().endObject()
                 .endObject().bytes();
 
-        ParseContext.Document doc = docMapper.parse("test", "type1", "1", json).rootDoc();
+        ParseContext.Document doc = docMapper.parse(SourceToParse.source("test", "type1", "1", json, 
+                XContentType.JSON))
```

Comment:
This one looks a bit ugly. I'd try moving `.rootDoc();` onto the line above and then fiddle with it. Not a big deal, just funky looking.